py/trtorch/__init__.py

```diff
@@ -9,6 +9,7 @@
 
 from trtorch._version import __version__
 from trtorch._compiler import *
+from trtorch.ptq import *
```

Comment:
Does this dump everything in the `trtorch` namespace? I would like to see the ptq stuff in its own subpackage